src/main/java/bot/listeners/MessageListener.java

```diff
@@ -23,7 +23,7 @@ public void onMessageReceived(MessageReceivedEvent event) {
         User author = event.getAuthor();
         Message message = event.getMessage();
         MessageChannel channel = event.getChannel();
-        String msg = message.getContent();
```

Comment:
`getContent` is no longer a method in newer versions of JDA.
The method has been split into `getContentDisplay` and `getContentRaw`